src/layouts/partial-cards.html

```diff
@@ -38,7 +38,7 @@
     </style>
     <app-route
       route="{{route}}"
-      pattern="/states/:view"
+      pattern="/:view"
```

Comment:
I think this is already working fine:
 - The element is hidden by `iron-pages` when not selected
 - Previously for `automation/edit` `routeMatch` was false causing  `currentView` to be "". Now it is passed in `statesRouteTail` that doesn't match `automation/edit` and `currentView` will still be "".